main.py

```diff
@@ -1,12 +1,21 @@
-"""ToDo."""
-from tasks.kyu8 import example
+# -*- coding: utf-8 -*-
+"""Python CodeWars task. 7-kyu
+        https://www.codewars.com/kata/sum-of-angles
+"""
+from tasks.kyu7 import angle
 
 
-def run_example():
-    """ToDo"""
-    first = input("a: ")
-    second = input("b: ")
-    print(example(first, second))
+def run_sum_of_angles():
+    """
+    Runs bin_to_decimal function.
+    """
+    number = 2
+    try:
+        number = int(input("number of angles: "))
+    except ValueError:
+        print("Invalid input. It should be int")
+    print(angle(int(number)))
```

Comment:
Why do you convert number to int twice? You may get an uncaught exception. Call the function in try-block

Reply:
Oh, it is a mistake, thanks, I've changed it